driver-kotlin-sync/src/integration/kotlin/com/mongodb/kotlin/client/UnifiedCrudTest.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.kotlin.client
+
+import java.io.IOException
+import java.net.URISyntaxException
+import org.bson.BsonArray
+import org.bson.BsonDocument
+import org.junit.Assume.assumeFalse
+import org.junit.runners.Parameterized
+
+internal class UnifiedCrudTest(
```

Comment:
I didn't mean that `CrudTest` should be deleted.  Just that we should also add this.  Can you re-add `CrudTest`?

Reply:
:facepalm: whoops - its back :)